library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -289,6 +294,45 @@ data class Conversations(
         return conversation
     }
 
+    fun getHmacKeys(
+        request: Keystore.GetConversationHmacKeysRequest? = null,
+    ): Keystore.GetConversationHmacKeysResponse {
+        val thirtyDayPeriodsSinceEpoch = (Date().time / 1000 / 60 / 60 / 24 / 30).toInt()
```

Comment:
is there a reason this is different than the previously used value?

```
Instant.now().epochSecond / 60 / 60 / 24 / 30
```

Reply:
I was just copying how you did it here: https://github.com/xmtp/xmtp-js/pull/519/files#diff-3f8d138711ab8a38267eef7baab8b362cbe3deee7fda5e18689bae9c2db74bb9R306